fhirResourcesToLoad/medication_methotrexate.json

```diff
@@ -0,0 +1,51 @@
+{
+    "resourceType": "Medication",
+    "id": "med-methotrexate",
+    "code": {
+      "coding": [
+        {
+          "system": "http://hl7.org/fhir/sid/ndc",
+          "code": "00555-0928-01",
+          "display": "Methotrexate"
+        }
+      ]
+    },
+    "form": {
+      "coding": [
+        {
+          "system": "http://snomed.info/sct",
+          "code": "385057009",
+          "display": "Film-coated tablet (qualifier value)"
+        }
+      ]
+    },
+    "ingredient": [
+      {
+        "itemCodeableConcept": {
+          "coding": [
+            {
+              "system": "http://snomed.info/sct",
+              "code": "776732000",
+              "display": "Product containing only methotrexate (medicinal product)"
+            }
+          ]
+        },
+        "strength": {
+          "numerator": {
+            "value": 7.5,
+            "system": "http://unitsofmeasure.org",
+            "code": "mg"
+          },
+          "denominator": {
+            "value": 1,
+            "system": "http://terminology.hl7.org/CodeSystem/v3-orderableDrugForm",
+            "code": "Tab"
+          }
+        }
+      }
+    ],
+    "batch": {
+      "lotNumber": "12345",
+      "expirationDate": "2026-06-23"
+    }
+  }
```

Comment:
Using FHIR validator, the following validation errors are found: 

Validation errors:
MedicationStatement: dom-3: 'If the resource is contained in another resource, it SHALL be referred to from elsewhere in the resource or SHALL refer to the containing resource' 'If the resource is contained in another resource, it SHALL be referred to from elsewhere in the resource or SHALL refer to the containing resource' ( (unmatched: med-methotrexate)) on line 1. Jump to error.
MedicationStatement.contained.ofType(Medication).ingredient[0].strength.denominator: Unknown Code http://terminology.hl7.org/CodeSystem/v3-orderableDrugForm#Tab in http://terminology.hl7.org/CodeSystem/v3-orderableDrugForm for 'http://terminology.hl7.org/CodeSystem/v3-orderableDrugForm#Tab' on line 44. Jump to error.